cult/templates/404.html

```diff
@@ -0,0 +1,12 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"inherit":true}} -->
+<main class="wp-block-group">
+
+	<!-- wp:pattern {"slug":"cult/404"} /-->
+	<!-- wp:search {"label":""} /-->
+
+</main>
+<!-- /wp:group -->
+
+<!-- wp:template-part {"slug":"footer","tagName":"footer","className":"site-footer-container"} /-->
```

Comment:
The source theme (and thus this one) have the 'site-footer-container' class throughout and isn't used for anything.  Suggest removing it from here (and Block Canvas).

Reply:
yes, let's do that, nice catch!